index.html

```diff
@@ -47,11 +47,9 @@ <h4>Compiler Migration Deemed Sufficiently Complete!</h4>
                 <p>Ping "@conda-forge/core" as always, if you need help.
                 Thanks for your patience!</p>
                 <!-- Uncomment the following if you want the migration bar --->
-                <div id="migratorDiv" style="text-align:center;">
-                    <canvas id="migratorCanvas" style="background:white;"></canvas>
-                    <legend for="migratorCanvas"></legend>
-                    <script type="text/javascript" src="migrator.js"></script>
-                </div>
+				<div id="migratorDiv" style="text-align:center;">
+				</div>
+				<script type="text/javascript" src="migrator.js"></script>
```

Comment:
Why is this indented so much and why is the script outside of the div?

Reply:
Also, if you modify the index for testing, you need to modify `template.html` for testing

Reply:
The script is outside the div I think because it is modifying it? I did this with @mariusvniekerk so he might know.